apps/protocol-frontend/src/utils/web3.ts

```diff
@@ -27,7 +27,7 @@ const gnosisChain: Chain = {
 
 const dev = import.meta.env.MODE || false;
 const defaultChains = dev
-  ? [gnosisChain, chain.goerli, chain.rinkeby, chain.localhost]
+  ? [gnosisChain, chain.mainnet, chain.goerli, chain.rinkeby, chain.localhost]
```

Comment:
Don't we need mainnet when dev is false?

Reply:
Should also remove it from dev if we aren't supporting

Reply:
I added in the `chain.mainnet` in the prior version of this PR when we used the wagmi hook for ENS. Removing this won't cause confusion in the interim and still lets use ENS